web-client/src/views/StartCaseInternal/OrdersNeededSummary.jsx

```diff
@@ -2,7 +2,7 @@ import { Button } from '../../ustc-ui/Button/Button';
 import React, { useEffect, useRef, useState } from 'react';
 import classNames from 'classnames';
 
-export const OrdersNeededSummary = ({ caseInformation }) => {
+export const OrdersNeededSummary = ({ reviewSavedPetitionHelper }) => {
```

Comment:
Is there a reason to pass in this helper instead of retrieving it off state?

Reply:
No particular reason at all. Just following the previous pattern. We can make that suggested adjustment. No worries

Reply:
Refactor completed.